src/utilities/hypre_cuda_utils.h

```diff
@@ -664,6 +664,19 @@ struct less_than : public thrust::unary_function<T,bool>
    }
 };
 
+template<typename T>
```

Comment:
Should this be a '.hpp' file?

Reply:
@rfalgout In this `.h`, we put all kinds of cuda C++ related stuff, which will be dumped into `_hypre_utilities.hpp` along with 2 other C++ `.h`. Would you like to rename all with `.hpp`? Thanks!

Reply:
If they are all getting dumped into `_hypre_utilities.hpp` and they contain C++ stuff, then I think it makes sense to rename all three of these files with `.hpp` extensions.  This is minor, though, and not crucial to getting this pull request and release done; we can worry about it later.

Reply:
> If they are all getting dumped into `_hypre_utilities.hpp` and they contain C++ stuff, then I think it makes sense to rename all three of these files with `.hpp` extensions. This is minor, though, and not crucial to getting this pull request and release done; we can worry about it later.

Sure. No problem. Will do it later. Thanks!